Makefile

```diff
@@ -34,3 +34,6 @@ ifeq ($(NOSTRETCH), 0)
 	BLDENV=stretch make -f Makefile.work $@
 endif
 	BLDENV=buster make -f Makefile.work $@
+
+freeze:
+	@scripts/versions_manager.py freeze $(OPTIONS)
```

Comment:
>OPTIONS [](start = 39, length = 7)

OPTIONS seems a name too general.
Also why not add it into rule/config, and provide comments which are the options for `OPTIONS`?

Reply:
The comments added, it is a command line to freeze versions. for most cases we can simple call: make freeze
For more options, we can run "scripts/versions_manager.py freeze -h" for help.

Reply:
The comments added, it is a command line to freeze versions. for most cases we can simple call: make freeze
For more options, we can run "scripts/versions_manager.py freeze -h" for help.

---
In reply to: [526567517](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r526567517) [](ancestors = 526567517)

Reply:
How about `FREEZE_VERSION_OPTIONS`?

---
In reply to: [526567517](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r526567517) [](ancestors = 526567517)

Reply:
Sure, I will fix it, thanks.

---
In reply to: [532347772](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r532347772) [](ancestors = 532347772,526567517)